### DISCUSSION TITLE
Beatmap offset slider may randomly jump back after moving it

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#17110] [Q&A] [answered]
very jank

it's only this slider as far as I know

https://user-images.githubusercontent.com/36758269/156895638-480d958a-685f-489b-a0d2-2be82a26f499.mp4

[logs.zip](https://github.com/ppy/osu/files/8191222/logs.zip)

Version: 2022.306.0-lazer

**Comment by peppy** [comment, marked as answer]
Looks like feedback from the realm event binding. A bit unfortunate.

**Reply by peppy** [reply]
Tracking in #17141.
